lib/oli/delivery/sections.ex

```diff
@@ -4952,50 +4952,6 @@ defmodule Oli.Delivery.Sections do
     end)
   end
 
-  @doc """
-  Maps each resource with its parent container label, being the label (if any) like
-  <Container Label> <Numbering Index>: <Container Title>
-
-  For example:
-
-  %{1: "Unit 1: Basics", 15: nil, 45: "Module 3: Enumerables"}
-  """
-  def map_resources_with_container_labels(section_slug, resource_ids) do
```

Comment:
This was the second query that was run to load each tab.  We no longer need it as we can glean the same information from `SectionResourceDepot`